src/net/sourceforge/kolmafia/request/GenericRequest.java

```diff
@@ -1393,15 +1393,7 @@ public void externalExecute() {
 
   public static final boolean shouldIgnore(final GenericRequest request) {
     String requestURL = GenericRequest.decodeField(request.formURLString);
-    return requestURL == null
-        ||
-        // Disallow mall searches
-        requestURL.contains("mall.php")
-        || requestURL.contains("manageprices.php")
-        || requestURL.contains("backoffice.php")
-        ||
-        // Disallow anything to do with chat
-        request.isChatRequest;
+    return false;
```

Comment:
is this change related, or has it snuck in?

Reply:
One day I'll stop accidentally adding this... test... code